Xamarin.Forms.Platform.Android/EmbeddedFontLoader.cs

```diff
@@ -14,7 +14,8 @@ public class EmbeddedFontLoader : IEmbeddedFontLoader
 		{
 			var tmpdir = IOPath.GetTempPath();
 			var filePath = IOPath.Combine(tmpdir, font.FontName);
-			if (File.Exists(filePath))
+			var fileInfo = new FileInfo(filePath);
+			if (fileInfo.Exists && fileInfo.Length == font.ResourceStream.Length)
```

Comment:
Is using the Length really safe here? It could lead to false positives.

https://stackoverflow.com/questions/1358510/how-to-compare-2-files-fast-using-net
https://docs.microsoft.com/en-us/troubleshoot/dotnet/csharp/create-file-compare

Reply:
Yes, this could lead to false positives. Better solution is to create a hash, but i think in most cases it will work.
So i'm not sure what the best/fastest way to implement it in xamarin android. Feel free to create another PR to solve this issue.